### PR TITLE
MRESOLVER-210 - Use try with resource

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/main/java/org/apache/maven/resolver/internal/ant/AntRepoSys.java [hunk 1]
```diff
@@ -821,17 +821,13 @@ private void populateCollectRequest( CollectRequest collectRequest, Task task, R
...
                     dependencies.add( dependency );
                 }
             }
-            finally
-            {
-                reader.close();
-            }
         }
-        catch ( IOException e )
+        catch ( final IOException e )
```
Comment (khmarbaise) [inline src/main/java/org/apache/maven/resolver/internal/ant/AntRepoSys.java:830]: Why this change? The ticket describes changes related to try-with-resource?

Comment (arturobernalg) [reply src/main/java/org/apache/maven/resolver/internal/ant/AntRepoSys.java:830]: HI @arturobernalg 
Correct. The main change it's use try-with-resource. Just a improve of the same code

Comment (khmarbaise) [reply src/main/java/org/apache/maven/resolver/internal/ant/AntRepoSys.java:830]: That's simply not related and shouldn't being done with the same issue. I would suggest to focus on the subject of the issue...

Comment (arturobernalg) [reply src/main/java/org/apache/maven/resolver/internal/ant/AntRepoSys.java]: Reverted all the secondary micro changes.
TY

**Comment by khmarbaise** [comment]
Has been merged into master https://github.com/apache/maven-resolver-ant-tasks/commit/8d4010e7686f193f3b18fd074eb71d4cb6e81b81
Thanks for your contribution.
